README.md

```diff
@@ -24,10 +24,9 @@ of these services, you likely would not use a docker version of them in producti
 - If this fails, then manual configuration is necessary before compilation. Perform the following:
    1. `docker network create nginx-proxy`
    2. `docker network create st-mariadb-102`
```

Comment:
Are allt hese networks unneeded now?

Reply:
For new dbs yes, I could do a large breaking change and opt all the others to internal but it would require project updates for all.

Reply:
Maybe this release should add them both networks now, and we can tell people to update the projects... then, the next change, remove the networks. Something can be on two networks, right?

Reply:
That does work. I've introduced this here to review separately (along w/ feature) - https://github.com/sourcetoad/DevopsToolKit/pull/11

I can then merge it into this.